origin-dapp/src/components/listings-grid.js

```diff
@@ -29,50 +29,22 @@ class ListingsGrid extends Component {
   }
 
   render() {
-    const { contractFound, listingIds, search, featuredListingIds, hiddenListingIds } = this.props
-    let activePage, currentPageListingIds, includedFeaturedListingIds, listingIdsNotHidden, resultsCount
+    const { contractFound, listingIds, search, featuredListingIds = [] } = this.props
```

Comment:
Can we remove featuredListingIds ? Looks like it is only used for passing the "featured" property to the ListingCard, but the ListingCard should be able to get that from the listing data itself so the "featured" property could be removed ?

Reply:
Yes good point, leaving it to the listing card to get featured information.